2-mandatory/4-stocks.js

```diff
@@ -47,8 +71,16 @@ function getAveragePrices(closingPricesForAllStocks) {
                 (Apple's price on the 5th day) - (Apple's price on the 1st day) = 172.99 - 179.19 = -6.2
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
-function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+
+function getPriceChanges(CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS) {
```

Comment:
In the parameter of a function, we use lowercase names. Uppercase is widely used to describe a global value. Also, we have a global variable with the same name in this file `CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS` which want to avoid these type of ambiguities when we select a name.